dist/pf-relative.js

```diff
@@ -12,6 +12,14 @@ angular.module('pf-relative', [])
     var relative = new RelativeDate(scope.date);
     element.attr('datetime', scope.date);
     relative.onTick(element.text.bind(element));
+
+    scope.$watch('date', function (newDate) {
+      relative.setDate(newDate);
+    });
+
+    element.on('$destroy', function() {
```

Comment:
Same as the previous comment... it would be better if we can get rid of the anonymous function.